.gitignore

```diff
@@ -139,6 +139,22 @@ uk.*/
 uk.*/
 za.*/
 
+!providers/*/*/*/be.*/
```

Comment:
You can also just add a `/` before all the `<tld>.*/`s in the above block.

The `/` limits the pattern to only matching files/folders in the root of the git directory.